pkg/controllers/context/context_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Karmada Authors.
+Copyright 2024 The Karmada Authors.
```

Comment:
This is the creation time of the file. we do not need to change it.

Reply:
Thanks, i'll update the commit.